index.html

```diff
@@ -274,6 +274,321 @@ <h3 class="text-2 text-center color-accent">
       </div>
     </section>
 
+    <section class="out-production">
+      <div class="container">
+        <div class="row">
+          <div class="col">
+            <p class="text-2 text-center text-uppercase">наша продукція</p>
+          </div>
+        </div>
+
+        <div class="row">
+          <div class="col-12 col-lg-4 mx-auto mb-5">
+            <div class="out-production-block w-100 d-flex flex-column">
+              <img
+                src="src/assets/img/gorbusha-225.png"
+                alt="caviar"
+                class=""
+              />
+
+              <p class="text-4 text-center text-uppercase mt-3 mb-3">
+                Ікра горбуші
+              </p>
+              <p class="gilroy">
+                <span class="fw-bold">Склад:</span> ікра горбуші, сіль, олія<br /><br />
+                <span class="fw-bold">Тара:</span> скло<br /><br />
+                <span class="fw-bold">Колір:</span>
+                яскраво-жовтогарячий<br /><br />
+                <span class="fw-bold">Розмір ікринок:</span> 4 мм у діаметрі<br /><br />
+                <span class="fw-bold">Консистенція:</span> суха. Не більше ніж
+                5-7% рідини<br /><br />
+                <span class="fw-bold">Термін придатності до споживання:</span>
+                4 міс. у закритому вигляді, 2 тижні у відкритому
+              </p>
+              <div
+                class="out-production-block-weight d-flex w-100 align-items-center"
+              >
+                <div class="w-100">
+                  <p class="gilroy text-7 mb-0">
+                    <span class="text-6 fw-semibold mr-4">Вага</span> (нетто)
+                  </p>
+                </div>
+                <div class="dropdown">
+                  <a
+                    class="btn btn-btn-outline-dark dropdown-toggle out-production-block-gramm gilroy fw-semibold text-4"
```

Comment:
If you have a custom class(here `out-production-block-gramm`), write his last or first for all tags.